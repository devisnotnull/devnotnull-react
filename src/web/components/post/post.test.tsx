import React from 'react';
import { shallow } from 'enzyme';

import Post from './postBlurb';

describe('<Post />', () => {
  it('render Post Component', () => {
    const wrapper = shallow(
      <Post
        metadata={{
          title: '',
          summary: '',
          blurb: '',
          favicon: {},
          primaryImage: {},
          secondaryImage: {}
        }}
      ></Post>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
