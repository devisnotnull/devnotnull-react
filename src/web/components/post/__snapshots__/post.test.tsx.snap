// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Post /> render Post Component 1`] = `ShallowWrapper {}`;
